docs/public/sandbox.js

```diff
@@ -0,0 +1,19 @@
+import * as R from "https://esm.sh/remeda";
+
+window.R = R;
+
+const sansSerif = "font-family: sans-serif; font-size: 16px";
+const monospace = "font-family: monospace; font-size: 16px";
+
+console.log(
+  "%cYou can use Remeda with the %cR%c global.",
```

Comment:
instead of use, let's use "try out", the goal here is to provide a way to play around with Remeda, not a way to use it.
```suggestion
  "%cYou can try out Remeda right here with the %cR%c global.",
```